rfc021-platform-support-policy.md

```diff
@@ -48,14 +48,11 @@ Ubuntu Linux | 12.04, 14.04 | x86, x86_64 | deb | Ubuntu 12.04
 
 Tier 2 supported platforms are those on which Omnitruck will serve packages, but those packages may not have been built on that OS variant. Additionally, we may or may not do post-build verification on these platforms. A platform must have at least one Maintainer before being added to this tier.
 
-* SUSE Linux Enterprise Server 10, 11
+* SUSE Linux Enterprise Server 11, 12
 * Scientific Linux 5.x, 6.x and 7.x (i386 and x86-64)
-* Gentoo Linux (rolling release)
-* Arch Linux (rolling release)
 * Fedora (current non-EOL revisions)
-* OpenSUSE 13.1/13.2
+* OpenSUSE 13.1/13.2/42.1
```

Comment:
Since it's a hybrid of Suse and Opensuse we went with identifying it as opensuseleap in ohai. It's still suse platform family though. We don't yet have a bento image so that keeps us from testing cookbooks.